### DISCUSSION TITLE
How to uninstall?

### DISCUSSION THREAD

**Discussion by ayyneya** [TabbyML/tabby#1360] [Q&A] [answered]
Excuse my noob question, how do you uninstall everything that tabby installed (models etc), using brew uninstall just removes 50mb.

**Comment by wsxiaoys** [comment, marked as answer]
You can simply remove the `~/.tabby` directory, where all runtime data files are stored.

**Comment by ayyneya** [comment]
> You can simply remove the `~/.tabby` directory, where all runtime data files are stored.

Appreciate you thank you!
